libs/domain/product/images/src/images.styles.ts

```diff
@@ -0,0 +1,100 @@
+import { css } from 'lit';
+
+export const productImageStyles = css`
+  :host {
+    --gap: 15px;
+
+    display: block;
+  }
+
+  :host > oryx-layout {
+    display: grid;
+    gap: var(--gap);
```

Comment:
```suggestion
    gap: var(--gap, 15px);
```
And drop css var definition above

Reply:
That was not working, but I've changed a small rule in the layout styles to allow for this.